RockPaperScissors/RockPaperScissors/RockPaperScissors.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  RockPaperScissors.swift
+//  RockPaperScissors
+//
+//  Created by 박종화 on 2023/05/03.
+//
+
+struct RockPaperScissors {
+    var isFirstGame: Bool = true
+    var isSecondGame: Bool = false
+    var turnDictionary: [Bool: String] = [true: "사용자", false: "컴퓨터"]
```

Comment:
딕셔너리를 사용해서 출력할 문자열을 관리하고 있군요 👍
문자열을 관리해주는 시도 너무 좋습니다! 
하지만 딕셔너리의 key값이 Bool타입인데 Bool타입 프로퍼티를 통해 무언가 제어하고 있는 상황이 아니기 때문에 문자열을 반환하는 작업이 조금 돌아가는 느낌이 드는 것 같습니다.

만약 `isUserTurn`이라는 Bool 타입 프로퍼티가 있다고 가정하면 isUserTurn을 딕셔너리의 키값으로 활용해볼 수 있겠네요.
```suggestion
    var isUserTurn: Bool = true
    var turnDictionary: [Bool: String] = [true: "사용자", false: "컴퓨터"]
    
    // ... 메서드 내부
        self.whoseTurn = turnDictionary[isUserTurn]
```


다만 연산프로퍼티를 활용하면 옵셔널을 사용하지 않을 수 있습니다.

```suggestion
    var isUserTurn: Bool = true
    var turn: String {
       return isUserTurn ? "사용자" : "컴퓨터"
     }
```

Reply:
true, false가 명시하는 것이 무엇인지 보여주기 위해 isUserTurn, isComputerTurn 불리언 변수로 각각 지정하여 수정했습니다.